src/crimson/mon/MonClient.cc

```diff
@@ -976,8 +976,9 @@ seastar::future<> Client::reopen_session(int rank)
     }).then([peer, this](auto result) {
       if (result == Connection::AuthResult::canceled) {
 	return seastar::make_exception_future(
-	    crimson::net::make_error_code(
-	      crimson::net::error::network_aborted));
+	    std::system_error(
+	      crimson::net::make_error_code(
+		crimson::net::error::network_aborted)));
```

Comment:
Not related to commit "crimson/net: wait for wandering connections to close when shutting down"